### PR TITLE
Added Brunt config flow

### PR DESCRIPTION
**Pull request by eavanvalkenburg**
## Proposed change

Added the config flow into the docs for Brunt.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/49714
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/brunt.markdown [hunk 1]
```diff
@@ -10,27 +10,16 @@ ha_codeowners:
...
-    password: BRUNT_PASSWORD
-```
-
-{% configuration %}
-name:
-  description: Cover name
-  required: false
-  default: "brunt blind engine"
-  type: string
+{% configuration_basic %}
```
Comment (MartinHjelmare) [inline source/_integrations/brunt.markdown]: I think we normally remove the YAML config docs when adding config flow and deprecating YAML.

Comment (eavanvalkenburg) [reply source/_integrations/brunt.markdown]: yeah, wasn't sure, deleted now.

File: source/_integrations/brunt.markdown [hunk 2]
```diff
@@ -10,36 +10,14 @@ ha_codeowners:
 ha_domain: brunt
 ha_platforms:
   - cover
+ha_config_flow: true
 ---
 
 The `brunt` platform allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use this sensor, you need a Brunt App Account. All Brunt Blind devices registered to your account are automatically added to your Home Assistant with the names given them through the Brunt app.
 
 ## Configuration
```
Comment (MartinHjelmare) [inline source/_integrations/brunt.markdown]: Check the preview build linked in the CI checks. This header is a duplicate. It's already included by the include below.

File: source/_integrations/brunt.markdown [hunk 3]
```diff
@@ -10,37 +10,13 @@ ha_codeowners:
 ha_domain: brunt
 ha_platforms:
   - cover
+ha_config_flow: true
 ---
 
 The `brunt` platform allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use this sensor, you need a Brunt App Account. All Brunt Blind devices registered to your account are automatically added to your Home Assistant with the names given them through the Brunt app.
```
Comment (MartinHjelmare) [inline source/_integrations/brunt.markdown]:
```suggestion
The `brunt` integration allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use it, you need a Brunt App Account. All Brunt Blind devices registered to your account are automatically added to your Home Assistant with the names given them through the Brunt app.
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**